src/app/zap-templates/zcl/data-model/chip/matter-devices.xml

```diff
@@ -1565,48 +1565,12 @@ limitations under the License.
                 <requireAttribute>CLIENT_LIST</requireAttribute>
                 <requireAttribute>PARTS_LIST</requireAttribute>
             </include>
-            <include cluster="Binding" client="true" server="false" clientLocked="false" serverLocked="true">
+            <include cluster="Binding" client="false" server="true" clientLocked="true" serverLocked="true">
```

Comment:
Door lock does not have a required Binding cluster.  I stopped here, for what it's worth.

Pease check very carefully that this PR is actually aligned with the spec, because reviewing this is _very_ time-consuming, and doing it multiple times is doubly so.  Again, I would urge doing one PR per device type, which would reduce the amount of re-review that needs to happen.